src/endpoints/getPlayerRanking.ts

```diff
@@ -1,18 +1,27 @@
+import querystring from 'querystring';
 import { PlayerRanking } from '../models/PlayerRanking'
 import { HLTVConfig } from '../config'
 import { fetchPage, toArray } from '../utils/mappers'
 
 export const getPlayerRanking = (config: HLTVConfig) => async ({
   startDate,
-  endDate
+  endDate,
+  matchType,
+  rankingFilter
 }: {
   startDate: string
   endDate: string
+  matchType: string
```

Comment:
An enum already exists for this: [MatchType](https://github.com/gigobyte/HLTV/blob/master/src/enums/MatchType.ts), it's better to use that instead of just string